src/main/java/com/github/seratch/jslack/Slack.java

```diff
@@ -63,6 +64,21 @@ public WebhookResponse send(String url, Payload payload) throws IOException {
                 .build();
     }
 
+    /**
+     * Send a data to Incoming Webhook endpoint.
+     */
+    public WebhookResponse send(String url, BlockMessagePayload payload) throws IOException {
```

Comment:
I don't see any necessity to have another payload class here. We can add the `blocks` property to the existing one. Also, updating the class/field comments would be preferable (if it's necessary).

Reply:
I've removed the `BlockMessagePayload` class and corresponding method